recovery.fstab

```diff
@@ -1,10 +0,0 @@
-/system       ext4          /dev/block/platform/soc.0/by-name/system
```

Comment:
https://github.com/boulzordev/android_device_lenovo_A6020/blob/cm-14.1/BoardConfig.mk#L119
check this